resources/security_policy.rb

```diff
@@ -1,14 +1,16 @@
 resource_name :security_policy
 
+default_action :configure
+
 property :policy_template, String, required: false, default: 'C:\Windows\security\templates\chefNewPolicy.inf'
 property :database, String, required: false, default: 'C:\Windows\security\database\chef.sdb'
 property :log_location, String, default: 'C:\Windows\security\logs\chef-secedit.log'
 
 action :configure do
   if node['platform'] == 'windows'
-    template "#{policy_template}" do
+    template '#{policy_template}' do
```

Comment:
```
inspec> policy_template="test"
=> "test"
inspec> '#{policy_template}'
=> "\#{policy_template}"
inspec> 
```
I am surprised, because in ruby you would need to use the double-quotes to get the variable in